api/share/responses.py

```diff
@@ -0,0 +1,37 @@
+from abc import ABC
```

Comment:
maybe this file should also be in a separate package for symmetry?

Reply:
It may not be suitable for this. Because we'll have a number of requsets (one requset corresponds to one use case) and only two responses - ResponseFailure and ResponseSuccess, which correspond to success or fail in user cases.

Reply:
OK got it